sbi/inference/snle/snle_base.py

```diff
@@ -79,10 +79,7 @@ def __init__(
         self._summary.update({"mcmc_times": []})  # type: ignore
 
     def append_simulations(
-        self,
-        theta: Tensor,
-        x: Tensor,
-        from_round: int = 0,
+        self, theta: Tensor, x: Tensor, from_round: int = 0,
```

Comment:
You're using an old `black` version. Please update and rerun. A trailing comma should trigger multi-line arguments

Reply:
will do